webpack.config.js

```diff
@@ -342,10 +373,15 @@ function getWebpackConfig( {
 			new WebpackRTLPlugin( {
 				minify: ! isDevelopment,
 			} ),
-			new AssetsWriter( {
-				filename: 'assets.json',
-				path: path.join( __dirname, 'server', 'bundler' ),
-			} ),
+			isCalypsoClient &&
+				new AssetsWriter( {
```

Comment:
Nice change! `AssetsWriter` is a Calypso specialty not needed anywhere else.

Reply:
Full credit goes to @simison on this one :)